src/Components/Pages/Home.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { AiOutlinePlus } from 'react-icons/ai';
+import Todo from '../Utilities/Todo';
+import Modal from '../Utilities/Modal';
+import '../Styles/Home.css';
+import { useModal } from '../Providers/ModalProvider';
+import { useTodos } from '../Providers/TodoProvider';
+
+const Home = () => {
+    const { modalOpen, setModalOpen } = useModal();
+    const { todos } = useTodos();
+    const [ rotation, setRotation ] = useState(0);
+    const [ searchQuery, setSearchQuery ] = useState('');
+
+    const toggleModal = e => {
+        setModalOpen(modalOpen.status === false ? {...modalOpen, status: true, data: { ...modalOpen.data, todoId: '', todoTitle: '', todoDesc: '', todoAlloted: '', todoTags: '', todoCreatedAt: ''  }} : {...modalOpen, status: false, data: { ...modalOpen.data, todoId: '', todoTitle: '', todoDesc: '', todoAlloted: '', todoTags: '', todoCreatedAt: ''  }})
+    }
+
+    useEffect(() => {
+        setRotation(modalOpen.status === true ? 45 : 0)
+    }, [modalOpen])
+
+    return (
+        <>
+            <nav className='navbar flex-center'>
+                <h1>TODOS</h1>
+                <input type='search' placeholder='Search by tags...' value={searchQuery} onChange={e => setSearchQuery(e.target.value)} />
```

Comment:
nice that you have used input type as "search". New thing to me

Reply:
thank you